packages/sourcecred/src/ui/components/LedgerViewer/LedgerEventRow.js

```diff
@@ -50,6 +62,17 @@ type LedgerEventRowProps = {|
 
 type EventRow = (LedgerEventRowProps) => ReactNode;
 
+function _friendlyFormatActionType(type: string): string {
+  //Split words by _, capitalize them, and join them with " "
+  var words = type.split("_");
+  words.forEach((word, idx) => {
+    word = word.toLowerCase();
+    word = word.charAt(0).toUpperCase() + word.slice(1);
+    words[idx] = word;
+  });
+  return words.join(" ");
+}
+
```

Comment:
I would put this in a `useCallback`

Reply:
Same question as above

Reply:
Actually this is fine it’s outside the component

Reply:
My bad